src/bridge/ExposureNotification.ts

```diff
@@ -19,6 +19,8 @@ export enum RiskLevel {
 }
 
 export enum Status {
+  // .Undefined is made up status to indicate js client that status hasn't been received from EN framework
+  Undefined = 'undefined',
```

Comment:
Didn't this get added in #153?

Reply:
yep, I'll need to rebase once #153 is merged.